redash/models/__init__.py

```diff
@@ -631,15 +631,18 @@ def fork(self, user):
         forked_list = ['org', 'data_source', 'latest_query_data', 'description',
                        'query_text', 'query_hash', 'options']
         kwargs = {a: getattr(self, a) for a in forked_list}
-        forked_query = Query.create(name=u'Copy of (#{}) {}'.format(self.id, self.name),
+
+        cls = type(self)
+        # Query.create will add default TABLE visualization, so use constructor to create bare copy of query
+        forked_query = cls(name=u'Copy of (#{}) {}'.format(self.id, self.name),
```

Comment:
```suggestion
        forked_query = Query(name=u'Copy of (#{}) {}'.format(self.id, self.name),
```

Why not use `Query` directly?

Reply:
@arikfr Thanks!. Done.